galaxy_ng/app/settings.py

```diff
@@ -453,3 +453,5 @@
 
 # feature flags
 FLAGS = {}
+
+SESSION_COOKIE_SECURE = os.environ.get('SESSION_COOKIE_SECURE', True)
```

Comment:
Envvars are strings, so this will evaluate to truthy even if someone exports `SESSION_COOKIE_SECURE=False`

This kind of problem is already handled by dynaconf on pulp.

You can simply default it to `True` and then use `PULP_SESSION_COOKIE_SECURE=true` on the envvar.


```suggestion
SESSION_COOKIE_SECURE = True
```